src/components/BankCard.tsx

```diff
@@ -9,6 +9,11 @@ type Props = {
 }
 
 function BankCard({ number, currency, amount, type }: Props) {
+  if (number?.charAt(0) === '4') {
```

Comment:
Remove if else construction and make a variable "type"
`const type = number?.charAt(0) === '4' ? 'Visa' : number?.charAt(0) === '5' ? 'MasterCard' : undefined`